### DISCUSSION TITLE
Can we sequence workflows based on attribute like correlationID

### DISCUSSION THREAD

**Discussion by bimal10** [Netflix/conductor#2996] [Ideas]
I see there is an option to sequence tasks within a workflow via FIFO queues. Also, tasktodomain allows us to run tasks in isolation, is there a similar feature available for workflows.

Usecase is to avoid parallel execution of a specific workflow so as to avoid race condition, especially in cases of updates. Ideally was thinking id correlation_id could be used for this very purpose. Any thoughts?

**Comment by apanicker-nflx** [comment]
Currently, conductor does not provide a mechanism to provide ordering across workflow executions.
